swap/src/cli/command.rs

```diff
@@ -765,6 +771,20 @@ mod tests {
         assert_eq!(args, Arguments::resume_testnet_defaults().with_debug());
     }
 
+    #[test]
+    fn given_help_does_not_panic() {
```

Comment:
I noticed that these tests may interrupt the test flow when run together with other tests. I guess this is because these new tests get interrupted before the function finishes.

Is there a better way to depict such kind of tests - i.e. check the exit code?

Reply:
Not that easy, maybe better to remove those tests. As far as I can see clap's [get_matches_from](https://docs.rs/clap/2.33.3/clap/struct.App.html#method.get_matches_from) calls [process::exit](https://doc.rust-lang.org/std/process/fn.exit.html) and we cannot check anything against that. Interesting that the test running gets interrupted by that.

Reply:
> Not that easy, maybe better to remove those tests. As far as I can see clap's get_matches_from calls process::exit and we cannot check anything against that. Interesting that the test running gets interrupted by that.

Well, calling `cargo test` builds "just another binary" that ends up being executed so if you call `process::exit` anywhere in there, the test suite just shuts down :)

Reply:
You can actually find that binary in your `target` directory and run it manually as well!

Reply:
You mean from within the rust test? Like run the binary in the test?

Reply:
No, I was more pointing out that the entire test suite of a crate in Rust just gets compiled to a binary with `fn main` that ends up calling all your test functions in sequence and printing out error reporting in between.

Hence, calling `process::exit` behaves as expected: It shuts down this binary in the middle of the test run.